packaging/debian/dotnet-debian_config.json

```diff
@@ -33,13 +33,17 @@
         "libcurl3" : {}
     },
 
+    "package_conflicts" : [
+        "dotnet-nightly"
```

Comment:
Talked offline. `-dev` is a commonly used convention in debian for shipping header files and libraries.

We will stick with `-nightly`